webpack.config.js

```diff
@@ -90,61 +104,82 @@ module.exports = function(env = {}) {
           test: /\.css$/,
           use: [
             MiniCssExtractPlugin.loader,
-            "css-loader",
+            { loader: "css-loader", options: { url: cssLoaderUrlFilter } },
             {
               loader: "less-loader",
               options: {
-                javascriptEnabled: true,
+                lessOptions: {
+                  javascriptEnabled: true,
+                },
               },
             },
           ],
         },
         {
           test: /\.woff(2)?(\?v=[0-9]\.[0-9]\.[0-9])?$/,
-          use: {
-            loader: "url-loader",
-            options: {
-              limit: 10000,
-              mimetype: "application/font-woff",
-            },
-          },
+          type: "asset",
+          parser: { dataUrlCondition: { maxSize: 10000 } },
+          // generator: { mimetype: "application/font-woff" },
```

Comment:
I didn't find a way to specify the mimetype for the webpack asset modules. I found this syntax in some webpack PR but it doesn't work. None of our .woff files seem to be below 10k anyways, though, so maybe we can simply remove this dataUrlCondition and always include them as resources :shrug: